File: health_project/lib/models/personal_dto.dart
class PersonalDTO {
  final int accountId;
  final String firstname;
  final String lastname;
  final int height;
  final double weight;
  final String familyHealth;
  final String healthDescription;
  final String gender;
  final String address;
  final String phone;

  const PersonalDTO({
    required this.accountId,
    required this.firstname,
    required this.lastname,
    required this.height,
    required this.weight,
    required this.familyHealth,
    required this.healthDescription,
    required this.gender,
    required this.address,
    required this.phone,
  });

  factory PersonalDTO.fromJson(Map<String, dynamic> json) {
    return PersonalDTO(
      accountId: json['accountId'],
      firstname: json['firstname'],
      lastname: json['lastname'],
      height: json['height'],
      weight: json['weight'],
      familyHealth: json['familyHealth'],
      healthDescription: json['healthDescription'],
      gender: json['gender'],
      address: json['address'],
      phone: json['phone'],
    );
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['accountId'] = this.accountId;
    data['firstname'] = this.firstname;
    data['lastname'] = this.lastname;
    data['height'] = this.height;
    data['weight'] = this.weight;
    data['familyHealth'] = this.familyHealth;
    data['healthDescription'] = this.healthDescription;
    data['gender'] = this.gender;
    data['address'] = this.address;
    data['phone'] = this.phone;
    return data;
  }
}

File: healt_project_server/src/main/java/com/dh/health/dto/PersonalDTO.java
package com.dh.health.dto;

import java.io.Serializable;


public class PersonalDTO implements Serializable{
	
	
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	private int accountId;
	private String firstname;
	private String lastname;
	private double height;
	private double weight;
	private String familyHealth;
	private String healthDescription;
	private String gender;
	private String address;
	private String phone;
	private String avatar;
	
	public PersonalDTO() {
		super();
	}
	
	public PersonalDTO(int accountId, String firstname, String lastname, double height, double weight,
			String familyHealth, String healthDescription, String gender, String address, String phone) {
		super();
		this.accountId = accountId;
		this.firstname = firstname;
		this.lastname = lastname;
		this.height = height;
		this.weight = weight;
		this.familyHealth = familyHealth;
		this.healthDescription = healthDescription;
		this.gender = gender;
		this.address = address;
		this.phone = phone;
	}
	
	public int getAccountId() {
		return accountId;
	}
	public void setAccountId(int accountId) {
		this.accountId = accountId;
	}
	public String getFirstname() {
		return firstname;
	}
	public void setFirstname(String firstname) {
		this.firstname = firstname;
	}
	public String getLastname() {
		return lastname;
	}
	public void setLastname(String lastname) {
		this.lastname = lastname;
	}
	public double getHeight() {
		return height;
	}
	public void setHeight(double height) {
		this.height = height;
	}
	public double getWeight() {
		return weight;
	}
	public void setWeight(double weight) {
		this.weight = weight;
	}
	public String getFamilyHealth() {
		return familyHealth;
	}
	public void setFamilyHealth(String familyHealth) {
		this.familyHealth = familyHealth;
	}
	public String getHealthDescription() {
		return healthDescription;
	}
	public void setHealthDescription(String healthDescription) {
		this.healthDescription = healthDescription;
	}
	public String getGender() {
		return gender;
	}
	public void setGender(String gender) {
		this.gender = gender;
	}
	public String getAddress() {
		return address;
	}
	public void setAddress(String address) {
		this.address = address;
	}
	public String getPhone() {
		return phone;
	}
	public void setPhone(String phone) {
		this.phone = phone;
	}

	public String getAvatar() {
		return avatar;
	}

	public void setAvatar(String avatar) {
		this.avatar = avatar;
	}
	

	
	
}
